test/unit/tech/html5.test.js

```diff
@@ -517,3 +517,20 @@ QUnit.test('Html5#reset calls Html5.resetMediaElement when called', function() {
 
   Html5.resetMediaElement = oldResetMedia;
 });
+
+QUnit.test('Exception in play promise should be caught', function() {
```

Comment:
I think this test should be wrapped in a condition to check for `Promise` support. When it gets merged and runs in a browser that doesn't support `Promise` it will likely blow up.